.github/ISSUE_TEMPLATE/model_request.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Model Request
+about: Request a model that does not currently exist in the ONNX model zoo.
+title: ''
+labels: 'model request'
```

Comment:
change the label name to model-request (this is based on the new guidelines which will be published soon for onnx repos https://github.com/onnx/onnx/pull/2835/files)